environments/staging/app-processes.yml

```diff
@@ -43,7 +43,7 @@ celery_processes:
     reminder_case_update_queue,reminder_queue:
       pooling: gevent
       concurrency: 2 
-    repeat_record_queue:
+    repeat_record_queue, repeat_record_datasource_queue:
```

Comment:
nit: The repeater is named `DataSourceRepeater`. What do you think of renaming this to `datasource_repeat_record_queue`?

Same for all other similar names in this PR.

Reply:
😅 I had it that way but decided to change it so that alphabetically, these two queues would be next to each other.

Reply:
Fair enough. It was only a nitpick.